proto/enums/enums.proto

```diff
@@ -99,6 +99,8 @@ enum LogLevel {
     PANIC = 5;
     // Trace Level
     TRACE = 6;
+    // Skaffold's standard output level
```

Comment:
I think the "Skaffold standard output" doesn't precisely communicate to the reader.
Could something like "User-visible output level" work better (prob ok to keep the enum as `STANDARD`)?

Reply:
Sounds good to me 👍🏼